src/index.ts

```diff
@@ -15,20 +15,24 @@ import cron from 'node-cron'
 const provider = getDefaultProvider(process.env.CHAIN_NETWORK)
 let wallet
 
-if (process.env.WALLET_MNEMONIC) {
-  wallet = Wallet.fromMnemonic(process.env.WALLET_MNEMONIC).connect(provider)
-} else if (process.env.WALLET_PRIVATE_KEY) {
+if (process.env.WALLET_PRIVATE_KEY) {
   wallet = new Wallet(process.env.WALLET_PRIVATE_KEY, provider)
+} else if (process.env.WALLET_MNEMONIC) {
+  wallet = Wallet.fromMnemonic(process.env.WALLET_MNEMONIC).connect(provider)
 } else {
   console.error('Failed to load wallet mnemonic or private key.')
   process.exit(1)
 }
 
-import ReleaseRewardTask from './src/tasks/ReleaseReward'
+import UpdateMinterRewardTask from './tasks/UpdateMinterReward'
+import UpdateAMMRewardTask from './tasks/UpdateAMMReward'
+
+cron.schedule(process.env.UPDATE_MINTER_REWARD_INTERVAL!, async () => UpdateMinterRewardTask(wallet))
 
-cron.schedule(process.env.RELEASE_REWARD_TASK_INTERVAL!, async () => ReleaseRewardTask(wallet))
+cron.schedule(process.env.UPDATE_AMM_REWARD_INTERVAL!, async () => UpdateAMMRewardTask(wallet))
 
 console.log(`Script is now running...`)
 console.table({
-  RELEASE_REWARD_TASK_INTERVAL: process.env.RELEASE_REWARD_TASK_INTERVAL,
+  UPDATE_MINTER_REWARD_INTERVAL: process.env.UPDATE_MINTER_REWARD_INTERVAL,
+  UPDATE_AMM_REWARD_INTERVAL: process.env.UPDATE_AMM_REWARD_INTERVAL,
```

Comment:
Do we need this comma?
```suggestion
  UPDATE_AMM_REWARD_INTERVAL: process.env.UPDATE_AMM_REWARD_INTERVAL
```

Reply:
The dangling comma is not required but as far as I know I have been following this as part of JS best practices: https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8

Please let me know you're thoughts on this :)